extensions/spotify/init.lua

```diff
@@ -125,4 +153,54 @@ function spotify.getCurrentTrack()
     return tell('name of the current track')
 end
 
+--- hs.spotify.getPlayerState()
+--- Function
+--- Gets the current state of Spotify
+---
+--- Parameters:
+---  * None
+---
+--- Returns:
+---  * A string containing one of hs.spotify.state_stopped, hs.spotify.state_paused, hs.spotify.state_playing
```

Comment:
I tend to prefer lists to be done with Markdown, so:

 * A string containing one of the following constants:
  * `hs.spotify.state_stopped`
  * `hs.spotify.state_paused`
  * `hs.spotify.state_playing`